index.js

```diff
@@ -40,41 +26,32 @@ app.get('/', (req, res) => {
 
 app.listen(3001, () => console.log('Node server running on port 3001'))
 
-function getConfigMap() {
-    return client.api.v1.namespaces('default')
-        .configmaps('saas-customers')
-        .get()
+
+// http client
+
+// TODO read from config map
+const baseURL = "http://35.226.186.70:15472";
+//const baseURL = "http://cn-crd-controller:15472";
+const urls = {
+    "createHub": baseURL + "/hub",
+    "deleteHub": baseURL + "/hub",
```

Comment:
since these routes are the same could we just have one route e.g. `crudHub`? Or do you think the routes might change in the future?